docs/scenes/get-started/set-up-your-environment/instructions/_androidStudioEnvironmentInstructions.mdx

```diff
@@ -0,0 +1,169 @@
+import { Step } from '~/ui/components/Step';
+import { Terminal } from '~/ui/components/Snippet';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+
+## Install Node.js, Watchman, and JDK
```

Comment:
```suggestion
## Install Watchman and JDK
```

We should avoid listing Node.js as an installation step here since this is covered in the previous page under system requirements: http://docs.expo.dev-pr-28511.s3-website-us-east-1.amazonaws.com/get-started/create-a-project/.

Originally, this instruction worked in a standalone guide but not a good experience from user's pov to find out on the next page (after reading system requirement) we are telling them to install Node.js with a version management tool. 

If we want to be a bit opinionated about it, we can say use a version management tool under system requirement or avoid it altogether.